cmd/git-sync/main.go

```diff
@@ -673,6 +673,18 @@ func setRepoReady() {
 	repoReady = true
 }
 
+// cleanupWorkTree() is used on failure to remove the worktree folder
```

Comment:
not only on failures, but also used to remove the old worktree after a successful sync.

Reply:
i thought line 835 in this CL was used for that.
Do we see old worktrees not being cleaned up during normal use ?

Reply:
This is about updating the comment to mention that this function is also used for removing old worktree folder.

Reply:
Line 717 cleans up the new worktree, which is used to guarantee the new workstree is empty.
Line 835 cleans up the old worktree, which is a normal cleanup. We should make the comment clear about the two use cases.

Reply:
```suggestion
// cleanupWorkTree() is used for removing new worktree folder if it already exists before creation, and removing old worktree folder 
```

Reply:
updated the comment.